src/p5.dance.js

```diff
@@ -1138,6 +1157,27 @@ module.exports = class DanceParty {
       title,
     };
 
+    let currentMeasure = Math.floor(this.getCurrentMeasure());
+    this.sprites_.forEach(sprite => {
+      if (sprite.alternatingMoveInfo) {
+        let alternatingMoveInfo = sprite.alternatingMoveInfo;
+        let val = Math.floor((currentMeasure - sprite.alternatingMoveInfo.start) / sprite.alternatingMoveInfo.cadence);
+        if (val % 2 == 0) {
```

Comment:
I'm not sure I understand what this 2 does?

Reply:
This is how I'm reducing the measure number to one bit so that we can alternate between two states.